src/Akka.Persistence.Sql/Extensions/ConnectionFactoryExtensions.cs

```diff
@@ -52,14 +52,9 @@ internal static async Task<T> ExecuteQueryWithTransactionAsync<T>(
             DbStateHolder state,
             Func<AkkaDataConnection, CancellationToken, Task<T>> handler)
         {
-            using var cts = CancellationTokenSource.CreateLinkedTokenSource(state.ShutdownToken);
-            {
-                cts.CancelAfter(state.QueryThrottleTimeout);
-                await state.QueryPermitter.Ask<QueryStartGranted>(RequestQueryStart.Instance, cts.Token);
-            }
```

Comment:
Remove complicated cancellation token code, replaced it with `Ask` internal timeout mechanism